lms/static/sass/discussion/_discussion.scss

```diff
@@ -4,17 +4,13 @@
 
 body.discussion {
 
-  .course-tabs .right {
-    @include float(right);
-  }
-
   .edit-post-form {
     @include clearfix();
     box-sizing: border-box;
     width: 100%;
 
     h1 {
-      font-size: 20px;
+      font-size: $forum-x-large-font-size;
```

Comment:
I think Andy answered this question in the discussion you guys already had, but let me know if not.